tests/userTests.py

```diff
@@ -158,12 +166,24 @@ def test_list_submissions(self):
         """Test listing user's submissions."""
         self.logout()
         self.login_approved_user()
-        response = self.app.get("/v1/list_submissions/", headers={"x-session-id":self.session_id})
+        response = self.app.get("/v1/list_submissions/", headers={"x-session-id": self.session_id})
         self.check_response(response, StatusCode.OK)
         self.assertIn("submission_id_list", response.json)
         self.assertEqual(len(response.json["submission_id_list"]), 5)
         self.logout()
 
+        self.login_agency_user()
+        response = self.app.get("/v1/list_submissions/", headers={"x-session-id": self.session_id})
+        self.check_response(response, StatusCode.OK)
+        self.assertIn("submission_id_list", response.json)
+        self.assertEqual(len(response.json["submission_id_list"]), 6)
+
+        response = self.app.get("/v1/list_submissions/?filter_by=agency", headers={"x-session-id": self.session_id})
+        self.check_response(response, StatusCode.OK)
+        self.assertIn("submissions", response.json)
+        self.assertEqual(len(response.json["submissions"]), 5)
```

Comment:
@nmonga91 Sorry for not noticing this yesterday, but I'm curious about why the format of the submission info being returned varies based on whether or not the  `filter_by_agency` is used? Would it make more sense for the route to return the submission info consistently?